### DISCUSSION TITLE
Incorrect characters in links

### DISCUSSION THREAD

**Discussion by matiechaza** [spatie/laravel-json-api-paginate#75] [Bugs]
When I get a paginated response, the package returns the link parameters as follows:
- **?page%5Bnumber%5D=1**

Instead of:
- **?page[number]=1**

Which causes me errors in certain cases. Is it a package error or a configuration that I am missing?
Thank you so much.

**Comment by AlexVanderbist** [comment]
I don't think this is the package returning these links. It's probably your browser URL encoding the `[` and `]` characters, as it should. However, this should work exactly the same way as the original URL.
